src/Unleash/Internal/UnleashServices.cs

```diff
@@ -84,6 +85,8 @@ public UnleashServices(UnleashSettings settings, EventCallbackConfig eventConfig
                 {
                     AppName = settings.AppName,
                     InstanceTag = settings.InstanceTag,
+                    ConnectionId = connectionId,
+                    SdkVersion = settings.SdkVersion,
```

Comment:
@kwasniew don't know if this answers your questions, but the "real" header value is read here from a default implementation in UnleashSettings: https://github.com/Unleash/unleash-client-dotnet/blob/main/src/Unleash/UnleashSettings.cs#L144C31-L144C44

Reply:
I'd fix the original versioning to follow unleash-client-dotnet:semver convention and drop the v prefix

Reply:
one drawback is that there's a lot of existing dotnet SDKs using the v version scheme